packages/ui-color-picker/src/ColorMixer/Slider/index.tsx

```diff
@@ -216,11 +216,10 @@ class Slider extends Component<SliderProps> {
         onMouseDown={(e) => this.handleMouseDown(e)}
         tabIndex={this.props.disabled ? undefined : 0}
         aria-label={this.props.navigationExplanationScreenReaderLabel}
-        // TODO: check RangeInput and ProgressBar to see how the slider role can be handled well
-        // role="slider"
-        // aria-valuemin={this.props.minValue}
-        // aria-valuemax={this.props.maxValue}
-        // aria-valuenow={this.roundedValue}
+        role="slider"
+        aria-valuemin={this.props.minValue}
+        aria-valuemax={this.props.maxValue}
+        aria-valuenow={this.roundedValue}
```

Comment:
This change is included in your other PR, https://github.com/instructure/instructure-ui/pull/1152/files#diff-298b371b157952c92f631f39b715112b2849486e6f3d34158127856e2d4ca0c0.